src/ontology/mondo-ingest.Makefile

```diff
@@ -581,6 +581,32 @@ $(REPORTDIR)/%.subclass.confirmed.robot.tsv $(REPORTDIR)/%.subclass.added.robot.
 	--mondo-mappings-path $(TMPDIR)/mondo.sssom.tsv \
 	--onto-config-path metadata/$*.yml
 
+.PHONY: sync-synonyms
+sync-synonyms: $(foreach n,$(ALL_COMPONENT_IDS), $(REPORTDIR)/$(n)-synonyms.added.robot.tsv)  # 'added' chosen arbitrarily as the 1 of 4 cases to run script for all sources
+
+tmp/mondo-synonyms-scope-type-xref.tsv: tmp/mondo.owl
+	$(ROBOT) query -i $< --query ../sparql/mondo-synonyms-scope-type-xref.sparql $@
+
+# todo: temp output for analysis during development
+INPUT_FILES := $(wildcard tmp/synonym_sync_combined_cases_*.tsv)
+tmp/synonym_sync_combined_cases.tsv: $(foreach n,$(ALL_COMPONENT_IDS), $(REPORTDIR)/$(n)-synonyms.added.robot.tsv)
+	@head -n 1 $(firstword $(INPUT_FILES)) > $@
+	@for file in $(INPUT_FILES); do \
+		tail -n +2 $$file >> $@; \
+	done
+
+$(REPORTDIR)/%-synonyms.added.robot.tsv $(REPORTDIR)/%-synonyms.confirmed.robot.tsv $(REPORTDIR)/%-synonyms.deleted.robot.tsv $(REPORTDIR)/%-synonyms.updated.robot.tsv: $(COMPONENTSDIR)/%.db metadata/%.yml tmp/mondo-synonyms-scope-type-xref.tsv
```

Comment:
## Output file names
- [ ] @twhetzel @matentzn Let me know if you would like this changed in any way, e.g. delimiters (`-`, `.`).

Example file name: `doid-synonyms.added.robot.tsv`